doc/source/examples/BasisSetBenchmark/BasisSetBenchmark.ipynb.rst

```diff
@@ -16,7 +16,7 @@ Initial Imports
 
 ::
 
-   PLAMS working folder: /path/plams/examples/BasisSetBenchmark/plams_workdir
```

Comment:
is this change intended?

Reply:
Consequence of re-running the notebook, have reverted.